templates/zerver/help/configure-who-can-create-streams.md

```diff
@@ -2,11 +2,19 @@
 
 {!admin-only.md!}
 
-Zulip allows you to separately control
-[permissions](/help/roles-and-permissions) for creating public and private
-streams. For corporations and other entities with controlled access, we highly
-recommend keeping stream creation open. Open organizations may choose to be less
-permissive, especially with public streams.
+Zulip allows you to separately control [permissions](/help/roles-and-permissions)
+for creating [web-public](/help/web-public-streams), public and private
+streams.
+
+For corporations and other entities with controlled access, we highly
+recommend keeping stream creation open. Open organizations may choose
+to be less permissive, especially with web-pubic and public streams.
+
+Only users in trusted roles like **Moderators** can be given permission
```

Comment:
I think we want to move away from capitalizing roles, and we probably don't need to bold it? Maybe:

> Only users in trusted roles (moderators and administrators) can be...

We should update the text on `web-public-streams` to match.

Reply:
Added that change to `web-public-streams` to the updates to this PR.